core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -25,161 +25,177 @@
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
+import org.apache.iceberg.BaseMetastoreTableOperations;
 import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.relocated.com.google.common.base.Joiner;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.base.Splitter;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 
 final class JdbcUtil {
   // property to control strict-mode (aka check if namespace exists when creating a table)
   static final String STRICT_MODE_PROPERTY = JdbcCatalog.PROPERTY_PREFIX + "strict-mode";
 
-  // Catalog Table
-  static final String CATALOG_TABLE_NAME = "iceberg_tables";
+  // Catalog Table & View
+  static final String CATALOG_TABLE_VIEW_NAME = "iceberg_tables_views";
```

Comment:
Is this safe to just rename? what happens if somebody has existing data in `iceberg_tables` and then upgrades the Iceberg version where `iceberg_tables_views` has been introduced?
Same comment applies to the rename from `table_namespace` to `namespace`.

Unless I'm missing something, I believe we need to keep the table names as-is and just add the new `type` column when the catalog initalizes its tables?

Reply:
Yes. Good point. I think it's better to use the previous table name and upgrade the schema. I plan to add a check in the initial method to check the current schema and update to the new one accordingly. 

Thanks for pointing this.

Reply:
True. I was giving the same comment. 
https://github.com/apache/iceberg/pull/9487#discussion_r1466256884

Race condition on giving comments :D